code/websites_and_search_engines.py

```diff
@@ -11,6 +11,11 @@
 )
 
 website_defaults = {
+    "talon website":"http://talonvoice.com",
```

Comment:
Technically all of these are websites.  Maybe
```suggestion
    "talon home page":"http://talonvoice.com",
```

Reply:
@rntz / @tararoys and one more?